contrib/asff.tpl

```diff
@@ -33,7 +33,7 @@
             "Severity": {
                 "Label": "{{ $severity }}"
             },
-            "Title": "Trivy found a vulnerability to {{ .VulnerabilityID }} in container {{ $target }}",
+            "Title": "Trivy found a vulnerability to {{ .VulnerabilityID }} in container {{ $target }}, related to {{ .PkgName }}",
```

Comment:
What about misconfigurations and secrets?
I thinks we can also add this change for them.

Reply:
What would you expect misconfigurations and secrets to look like? The reason that I am adding `PkgName` is because AWS essentially does a put command on the `Title` so if the `VulnerabiltyID` and `target` are the same for multiple packages then it only saves the latest. With secrets and misconfigurations the `.Title` is file specific which is detailed enough for me. I guess if you have multiple occurrences of the same misconfiguration type in one file then it would only upload one. With the fields provided the only thing I could think of to fix that would be the creation/update time

Reply:
Let me know if that makes sense. I'm not the most articulate 😅

Reply:
>  I guess if you have multiple occurrences of the same misconfiguration type in one file

I thought about same misconfiguretaions/secrets in different files.
e.g.
```Dockerfile
FROM alpine

RUN mkdir foo

RUN echo "AWS_ACCESS_KEY_ID=AKIA0123456789ABCDEF" > secret.txt

RUN echo "AWS_ACCESS_KEY_ID=AKIA0123456789ABCDEF" > foo/secret2.txt
``` 
```
➜ trivy image -f template -t @contrib/asff.tpl 3183 | grep Title -A 1
            "Title": "Trivy found a secret in /secret.txt: AWS Access Key ID",
            "Description": "Trivy found a secret in /secret.txt: AWS Access Key ID",
--
            "Title": "Trivy found a secret in /foo/secret2.txt: AWS Access Key ID",
            "Description": "Trivy found a secret in /foo/secret2.txt: AWS Access Key ID",
```
Or i am wrong and is AWS saving different files with same `Title` correctly?

Reply:
Okay, thanks for the explanation!